src/activities/ActivityUsageEntity.js

```diff
@@ -250,6 +250,20 @@ export class ActivityUsageEntity extends Entity {
 		activityCollectionHref && this._subEntity(ActivityUsageCollectionEntity, activityCollectionHref, onChange);
 	}
 
+	/**
+	 * @returns {bool} Whether or not the activity is reusable
+	 */
+	isReusableActivity() {
+		return this._entity.hasClass('reusable-activity');
```

Comment:
Consider:
Putting the string into the classes folder

Could this error if the entity is null?